Source/Utils/IntervalsParser.cpp

```diff
@@ -116,3 +122,74 @@ int IntervalsParser::localPeriod (const int n) const
 }
 
 bool IntervalsParser::isActivated () const {return m_activated;}
+
+BTDIntervalsParser::BTDIntervalsParser (const std::vector<std::string>& instr_vec)
+{
+    std::string inconcatenated;
```

Comment:
More of a C++ question, I guess: is `inconcatenated` always going to be initialized as an empty string `''` by default (which has to be the case, due to the upcoming `+=` assignements)?

Reply:
I assume so because that was copied from the IntervalsParser constructor but is not code I thought about